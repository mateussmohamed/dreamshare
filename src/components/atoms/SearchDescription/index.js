import styled from 'styled-components'

const SearchDescription = styled.h4`
  ${props => props.theme.typography.robotoStyle};
  font-size: 16px;
  text-align: center;
  color: ${props => props.theme.colors.brownishGrey};
`

export default SearchDescription
