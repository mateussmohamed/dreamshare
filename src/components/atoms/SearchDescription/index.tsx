import styled from 'styled-components'

export const SearchDescription = styled.h4`
  ${props => props.theme.typography.robotoStyle};
  font-size: 16px;
  text-align: center;
  color: ${props => props.theme.colors.brownishGrey};
`
