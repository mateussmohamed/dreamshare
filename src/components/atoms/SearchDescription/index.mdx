---
name: SearchDescription
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import SearchDescription from './index'

# SearchDescription
<PropsTable of={SearchDescription} />

<Playground>
  <SearchDescription>
    SHARE YOUR HOLIDAY DREAM
  </SearchDescription>
</Playground>
