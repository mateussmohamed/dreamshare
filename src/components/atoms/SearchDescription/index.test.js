import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import SearchDescription from './index'

describe('[<SearchDescription />]', () => {
  const wrap = (props = {}) => shallow(<SearchDescription {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a description' })

    expect(wrapper).toMatchSnapshot()
  })
})
