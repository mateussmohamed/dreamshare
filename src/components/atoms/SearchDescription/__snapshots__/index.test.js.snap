// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<SearchDescription />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "SearchDescription-sc-1t844l-0",
        "isStatic": false,
        "rules": Array [
          [Function],
          ";font-size:16px;text-align:center;color:",
          [Function],
          ";",
        ],
      },
      "displayName": "SearchDescription",
      "render": [Function],
      "styledComponentId": "SearchDescription-sc-1t844l-0",
      "target": "h4",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a description
</StyledComponent>
`;
